integration_test/nodes/docker-compose.yml

```diff
@@ -2,27 +2,27 @@ version: "3.9"
 
 services:
   l1_node:
-    image: ghcr.io/foundry-rs/foundry:latest
+    image: ghcr.dockerproxy.com/foundry-rs/foundry:latest
     restart: unless-stopped
     pull_policy: always
     ports:
-      - 18545:8545
-      - 18546:8545
+      - "18545:8545"
+      - "18546:8545"
     entrypoint:
       - anvil
       - --host
       - "0.0.0.0"
 
   l2_execution_engine:
-    image: gcr.io/evmchain/taiko-geth:taiko
+    image: gcr.dockerproxy.com/evmchain/taiko-geth:taiko
```

Comment:
Not sure if this will slow down those who are not in China?

Reply:
It's very fast not in China.